sdks/java/io/kudu/src/main/java/org/apache/beam/sdk/io/kudu/KuduIO.java

```diff
@@ -98,7 +98,7 @@
  * <h3>Writing to Kudu</h3>
  *
  * <p>The Kudu sink executes a set of operations on a single table. It takes as input a {@link
```

Comment:
Now there's a spotless violation for this paragraph.

Reply:
Fixed it. Thanks.